src/OSPSuite.Presentation/DTO/RenameObjectDTO.cs

```diff
@@ -7,6 +7,11 @@ public class RenameObjectDTO : ObjectBaseDTO
    {
       public string OriginalName { get; set; }
 
+      /// <summary>
+      /// Set to <c>true</c> (default), renaming a parameter using another case would be accepted. This is ok for rename scenario but not for clone scenario
+      /// </summary>
+      public bool AllowSameNameAsOriginalInDifferentCase { get; set; } = true;
```

Comment:
As I was testing, I realized it was possible to clone a PI and rename it using another case. It is not allowed for other builing block so this is a bug